GoogleForCreators/web-stories-wp
# Dashboard: Removed branded logos from templates and updated to latest data version

## Summary

This PR does two things:

1.)  It updates the data structure of the following templates to the latest version (v19):
- `DIY`
- `Cooking`
- `Entertainment`
- `Travel`
- `Wellbeing`

2.)  It removes the Bon Appetit logo from the `Cooking` template, and the Travel + Leisure logo from the `Travel` template.

## User-facing changes

`Cooking`: The first page of the Cooking template should have a white box instead of the "Bon Appetit" logo.
![image](https://user-images.githubusercontent.com/40646372/83443014-ec049c80-a3fd-11ea-9da7-cce2ab744c33.png)

`Travel`: The first page of the Travel template should have a white box instead of the "Travel+Leisure" logo.
![image](https://user-images.githubusercontent.com/40646372/83443092-0a6a9800-a3fe-11ea-91c0-795b51a5f5cb.png)


## Testing Instructions

Go to "Explore Templates" and make sure that the "Cooking" and "Travel" templates appear as pictured above.  And make sure that "DIY", "Entertainment", and "Wellbeing" appear without visual bugs (basically, they should look like they looked before, nothing changed on them but their underlying data structure).

---



Fixes #2159
